README.md

```diff
@@ -88,16 +88,16 @@ To get a book (in easy mode) we can filter by 4 fields (or a composition of thes
 * edition
```

Comment:
Acho que a frase acima poderia ser `To retrieve a book (in easy mode) we can filter by 4 fields (or a composition of these four):`